dolweb/downloads/templates/downloads-index.html

```diff
@@ -75,15 +75,15 @@
 <div id="download-dev">
 <h1>{% trans "Development versions" %}</h1>
 
-<div class="alert">{% blocktrans %}
+<p>{% blocktrans %}
```

Comment:
The `<p>` should likely be removed entirely, since the text is already in a `<p>`.